### PR TITLE
Buzzer Management

### PR DESCRIPTION
**Pull request by themacexpert**
Fix front end and back end (thanks @MJAlperin23)
Basic Structure for front end. 

screen shot 2016-11-03 at 4 49 58 pm

### CODE REVIEW

**Comment by MJAlperin23** [comment]
👎 devices is amisleadingg name

**Comment by MJAlperin23** [comment]
👎 devices is a misleading name

File: assets/templates/buzzer_management.html.tmpl [hunk 1]
```diff
@@ -1,41 +1,130 @@
...
+  <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+  <meta name="description" content="">
+  <meta name="author" content="">
+  <link rel="icon" href="../../favicon.ico">
+
+  <!-- Stuff for bootstrap table: http://www.w3schools.com/bootstrap/bootstrap_tables.asp-->
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+
+  <title>Buzzer - Waitlist</title>
```
Comment (jfeiber) [inline assets/templates/buzzer_management.html.tmpl]: probably shouldn't be waitlist

**Comment by MJAlperin23** [comment]
i would recommend closing this a restarting

**Comment by MJAlperin23** [comment]
👍

**Pull request merged**